Content.Server/Objectives/Components/DoorjackConditionComponent.cs

```diff
@@ -1,12 +1,15 @@
 using Content.Server.Objectives.Systems;
+using Content.Shared.Ninja.Systems;
 
 namespace Content.Server.Objectives.Components;
 
 /// <summary>
 /// Objective condition that requires the player to be a ninja and have doorjacked at least a random number of airlocks.
 /// Requires <see cref="NumberObjectiveComponent"/> to function.
 /// </summary>
-[RegisterComponent, Access(typeof(NinjaConditionsSystem))]
+[RegisterComponent, Access(typeof(NinjaConditionsSystem), typeof(SharedSpaceNinjaSystem))]
```

Comment:
add api to NinjaConditionsSystem dont spam access
then just _ninjaConditions.DoorJacked(obj) etc